corehq/apps/app_manager/static/app_manager/js/visit-scheduler.js

```diff
@@ -196,6 +379,40 @@ var VisitScheduler = (function () {
     };
 
     return {
-        Scheduler: Scheduler
+        Scheduler: Scheduler,
+        ModuleScheduler: ModuleScheduler
     };
 }());
+
+//connect items with observableArrays
+ko.bindingHandlers.sortableList = {
+    init: function(element, valueAccessor) {
+        var list = valueAccessor();
+        $(element).sortable({
+            update: function(event, ui) {
+                //retrieve our actual data item
+                var item = ko.dataFor(ui.item.get(0));
+                //figure out its new position
+                var position = ko.utils.arrayIndexOf(ui.item.parent().children(), ui.item[0]);
+                //remove the item and add it back in the right spot
+                if (position >= 0) {
```

Comment:
oh, i totally read that as `position > 0` (even when copying & pasting), sorry for the derail